packages/babel-preset-es2015/src/index.js

```diff
@@ -1,42 +1,50 @@
 module.exports = function(context, opts) {
-  /* eslint-disable no-var */
-  var loose = false;
-  var modules = true;
-  /* eslint-enable no-var */
+  const moduleTypes = ["commonjs", "amd", "umd", "systemjs"];
+  let loose = false;
+  let modules = "commonjs";
+  let spec = false;
```

Comment:
I suggested it for completeness. It seems like the sort of thing that makes sense to configure at the level of the `es2015` preset.